doc OKTA-106836

mystiberry-okta opened the pull request:
Needs review, especially the sample request and response I added. Used postman to create it.

mystiberry-okta commented:
@johngronberg-okta The process folks have asked that we use the Review feature for doc. In the following view, click the green Review Changes button:

https://github.com/okta/okta.github.io/pull/506/commits/347c52a2f0bb108cb2dd3387aa67f7b90ee7c778

mystiberry-okta commented:
Oops, found a missing bracket.

johngronberg-okta commented on _source/_docs/api/resources/authn.md:
```diff
@@ -56,6 +56,46 @@ The [transaction state](#transaction-state) of the response will depend on the u
 
+#### Example Request
+
+Request: `http://example.okta.com/api/v1/authn`
```
Should this say "endpoint" or something like that?

mystiberry-okta replied:
I got rid of it, as there's a far better example of the same thing. Hoping the links at the top of the section will do. Otherwise the structure of this doc gets really goopy.

johngronberg-okta commented on _source/_docs/api/resources/authn.md:
```diff
@@ -56,6 +56,46 @@ The [transaction state](#transaction-state) of the response will depend on the u
 
 > You must first enable MFA factors and assign a valid **Sign-On Policy** to a user to enroll and/or verify a MFA factor during authentication
 
+#### Example Request
```
Maybe add a description like "this is an example request for primary authentication with a public application"

mystiberry-okta replied:
There's already such an example in the doc. I removed my lesser example and made it explicit where to find the examples with links.

johngronberg-okta commented on _source/_docs/api/resources/authn.md:
```diff
@@ -607,8 +647,8 @@ curl -v -X POST \
-> The `warnBeforePasswordExpired` option must be explicitly specified as `true` to allow the authentication transaction to transition to `PASSWORD_WARN` status.<br>
-> Non-expired passwords will successfully complete the authentication transaction if this option is omitted or `false`.
+> * The `warnBeforePasswordExpired` option must be explicitly specified as `true` to allow the authentication transaction to transition to `PASSWORD_WARN` status.
+  * Non-expired passwords will successfully complete the authentication transaction if this option is omitted or `false`.
```
The nesting here does not look right
<img width="791" alt="screen shot 2016-11-23 at 3 33 47 pm" src="https://cloud.githubusercontent.com/assets/22499867/20582046/373529d4-b192-11e6-97ae-843fafc80201.png">

mystiberry-okta replied:
Fixed.

johngronberg-okta commented on _source/_docs/api/resources/authn.md:
```diff
@@ -794,11 +834,16 @@ curl -v -X POST \
+
+You can enroll, activate, and verify factors using the `/api/v1/authn/factors` endpoint.
+
+> You can also enroll, manage, and verify factors outside the authentication context with [`/api/v1/users/:uid/factors/`](factors.html#factor-verification-operations).
```
is :uid what we use everywhere?

mystiberry-okta replied:
Yes, unfortunately markdown won't let me use italics inside the grave marks. I don't love the :uid notation but I'm not sure where it comes from.

mystiberry-okta commented:
@johngronberg-okta , I fixed a few empty-column tables, added links so we don't have to repeat the first example (my version wasn't as good as the version already in the doc),  so this probably needs one more look.
